codechain/main.rs

```diff
@@ -225,6 +225,11 @@ fn run_node(matches: ArgMatches) -> Result<(), String> {
         }
     };
 
+    // FIXME: Get snapshot root directory from config
+    // FIXME: Get snapshot period from genesis block
+    let snapshot_service = SnapshotService::new(client.client(), "snapshot".to_string(), 1 << 14);
```

Comment:
What does `1 << 14` mean?

Reply:
It's hardcoded constant, and it needs to be read from genesis.
That work will be done in next PR.

Reply:
Clarify the intention by introducing a named constant.